src/coffee/sync/utils/inventory.coffee

```diff
@@ -49,23 +49,23 @@ class InventoryUtils extends BaseUtils
   # diff - {Object} The result of diff from `jsondiffpatch`
   #
   # Returns {Array} The list of actions, or empty if there are none
-  actionsMapRestockableInDays: (diff) ->
+  actionsMapRestockableInDays: (diff, old_obj) ->
     actions = []
     if diff.restockableInDays
       if _.isArray(diff.restockableInDays)
         size = _.size(diff.restockableInDays)
         a =
           action: 'setRestockableInDays'
-        if size is 1
-          a.restockableInDays = diff.restockableInDays[0]
-        else if size is 2
+        if size is 2
```

Comment:
If you don't change the order of the if/else statement, the actual change is one line.